selfdrive/modeld/fill_model_msg.py

```diff
@@ -0,0 +1,172 @@
+# ruff: noqa: E701
```

Comment:
what's this for? noqa should be used extremely sparingly

Reply:
this is for in-line ifs (one-line-colon) in the fill functions. I think it helps with readability.